docs/dgfit/plot_results.rst

```diff
@@ -15,6 +15,271 @@ For example, to plot the default run results from the optimizer (= max prob):
 
     plot_dgfit dgfit_test_WD_best_optimizer.fits obsdata
 
+.. plot::
+
+    from dgfit.plotting.plot_dgfit import (plot_dgfit_sizedist,
+                                            plot_dgfit_extinction,
+                                            plot_dgfit_emission,
+                                            plot_dgfit_albedo,
+                                            plot_dgfit_g,
+                                            plot_dgfit_abundances)
+    from dgfit.obsdata import ObsData
+    import numpy as np
+    import matplotlib.pyplot as pyplot
+    import matplotlib
+    from astropy.io import fits
+
+    OD = ObsData("mw_rv31_obs.dat", path = "../../dgfit/data/mw_rv31/")
+    hdulist = fits.open("../../dgfit/data/mw_rv31/dgfit_test_WD_sizedist_best_optimizer.fits")
+
+    fontsize = 16
+    font = {"size": fontsize}
+    matplotlib.rc("font", **font)
+    matplotlib.rc("lines", linewidth=2)
+    matplotlib.rc("axes", linewidth=2)
+    matplotlib.rc("xtick.major", width=2)
+    matplotlib.rc("ytick.major", width=2)
+
+    fig, ax = pyplot.subplots(ncols=3, nrows=2, figsize=(15, 10))
+
+    plot_dgfit_sizedist(ax[0, 0], hdulist, fontsize=fontsize, plegend=True)
+    plot_dgfit_abundances(
+        ax[0, 1],
+        hdulist["ABUNDANCES"],
+        OD,
+        fontsize=fontsize,
+        color="r",
+        plegend=True,
+        plabel="Final",
+    )
+    plot_dgfit_extinction(ax[1, 0], hdulist["EXTINCTION"], OD, fontsize=fontsize)
+    plot_dgfit_emission(ax[0, 2], hdulist["EMISSION"], OD, fontsize=fontsize)
+    plot_dgfit_albedo(ax[1, 1], hdulist["ALBEDO"], OD, fontsize=fontsize)
+    plot_dgfit_g(ax[1, 2], hdulist["G"], OD, fontsize=fontsize)
+
+    ax[0, 0].set_ylim(1e-40, 3e-27)
+    pyplot.tight_layout()
+
+The figure should look like this.
```

Comment:
This line should be removed.  Not needed and made me think there was supposed to be a plot *after* this line.